tests/test_authentication.py

```diff
@@ -264,41 +263,39 @@ def test_no_next_param(self):
             data={"email": self.user.email, "password": self.password},
             org=self.factory.org,
         )
-        self.assertEqual(
-            response.location, "http://localhost/{}/".format(self.user.org.slug)
-        )
+        self.assertEqual(response.location, "/{}/".format(self.user.org.slug))
```

Comment:
It seems that the location value in the response after redirecting in the new version of Flask is different from the old version.

Reply:
I think this might be a werkzeug change. Anyway it looks fine. 👍